cmd/apic/template.go

```diff
@@ -74,7 +76,17 @@ func (v *templateVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 	options := template.Options{
 		Dir:     v.Dir,
 		APIFile: api.Name(),
-		Loader:  ioutil.ReadFile,
+		Loader: func(filename string) ([]byte, error) {
```

Comment:
Style: I would be keen to reduce our use of lambdas, would it be OK to declare this function outside? (esp. here it doesn't seem to hoist any local vars.)

Reply:
It hoists v, but I can make it first-class function. That being said, for short functions, I tend to prefer lambdas if they are used only once. Otherwise you have to spend more time hunting through code.

Reply:
That is a matter of taste, but I tend to not like lambdas because they appear unnamed in stack traces :)
I let you make the decision.